deno_webgpu/src/binding.rs

```diff
@@ -70,18 +70,15 @@ enum GpuSamplerBindingType {
 impl From<GpuSamplerBindingType> for wgpu_types::BindingType {
     fn from(binding_type: GpuSamplerBindingType) -> Self {
         match binding_type {
-            GpuSamplerBindingType::Filtering => wgpu_types::BindingType::Sampler {
-                filtering: true,
-                comparison: false,
-            },
-            GpuSamplerBindingType::NonFiltering => wgpu_types::BindingType::Sampler {
-                filtering: false,
-                comparison: false,
-            },
-            GpuSamplerBindingType::Comparison => wgpu_types::BindingType::Sampler {
-                filtering: true,
-                comparison: true,
-            },
+            GpuSamplerBindingType::Filtering => {
```

Comment:
with this PR, we shouldn't need `GpuSamplerBindingType` at all. Deno should just reuse `SamplerBindingType` instead directly